src/Controller/CocktailController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Controller;
+
+use App\Form\CocktailFormType;
+use App\Entity\CocktailName;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Request;
+
+class CocktailController extends AbstractController
+{
+    #[Route('/home', name: 'home')]
+    public function new(Request $request): Response
+    {
+        $cocktailName = new CocktailName();
+
+        $form = $this->createForm(CocktailFormType::class, $cocktailName);
+        $form->handleRequest($request);
+
+        if($form->isSubmitted() && $form->isValid()){
```

Comment:
Later you could add some validations so only string or numstring would be preferred. (add this as a comment with TODO in front, PHPStorm automatically detects todos).

Reply:
fixed